package.json

```diff
@@ -7,6 +7,7 @@
     "@reach/window-size": "^0.13.2",
     "@styled-icons/boxicons-regular": "^10.23.0",
     "@styled-icons/boxicons-solid": "^10.23.0",
+    "@styled-icons/fa-solid": "^10.26.0",
```

Comment:
We're using boxicons for all our icons, so try to find the equivalent in box icons (either regular or solid) and remove font awesome. Thanks!